CMakeLists.txt

```diff
@@ -408,11 +408,10 @@ enable_testing()
 option(BUILD_DOCUMENTATION "Build the Doxygen documentation." ON)
 option(BUILD_TESTS "Build tests." OFF)
 if (CMAKE_BUILD_TYPE STREQUAL "Debug")
-  set(DEFAULT_BUILD_DEBUG_UTILITIES ON)
+  set(BUILD_DEBUG_UTILITIES ON)
 else()
-  set(DEFAULT_BUILD_DEBUG_UTILITIES OFF)
+  set(BUILD_DEBUG_UTILITIES OFF)
 endif()
-option(BUILD_DEBUG_UTILITIES "Build debug utilities." DEFAULT_BUILD_DEBUG_UTILITIES)
```

Comment:
```cmake
set(BUILD_DEBUG_UTILITIES DEFAULT_BUILD_DEBUG_UTILITIES CACHE BOOL "Build debug utilities." FORCE)
```

This should overwrite the cache due to the `FORCE` keyword.

Reply:
Is this for those who already have a CMake cache? If so, I could `unset(CACHE)` the variable before calling `set()`, so that the option doesn't appear in their cache despite it having no effect.